Source/WebCore/style/StyleAdjuster.cpp

```diff
@@ -144,7 +144,10 @@ static DisplayType equivalentBlockDisplay(const RenderStyle& style)
         return DisplayType::Grid;
     case DisplayType::Ruby:
         return DisplayType::RubyBlock;
-
+    case DisplayType::Math:
+       return DisplayType::Inline;
+    case DisplayType::MathBlock:
+       return DisplayType::Inline;
```

Comment:
again, it should be a block type. so probably this should just be moved above (below  `case DisplayType::RubyBlock`) where we return unmodified block displays.